app/config/security.yml

```diff
@@ -1,10 +1,10 @@
 security:
     encoders:
-        TobiasOlry\TalklyBundle\Entity\User: plain
+        TobiasOlry\TalklyBundle\Entity\User: bcrypt
```

Comment:
Couldn't find anything about this with a quick search: is it possible to use php's password_hash / password_verify for this? It includes hashing algorithm and salt in the hash + will update automatically to newer/safer hashing algorithms when available.